libraries/chain/include/eosio/chain/unapplied_transaction_queue.hpp

```diff
@@ -35,6 +35,7 @@ struct unapplied_transaction {
    const fc::time_point           expiry;
    trx_enum_type                  trx_type = trx_enum_type::unknown;
    next_func_t                    next;
+   bool                           return_failure_trace;
```

Comment:
Should init to `false` and move above `next` as it bugs me that it is diff order than in args of functions.

Reply:
`git commit -m "Cater to heifner's irrational prejudices..."`

I mean uuuuh....

`git commit -m "Cater to heifner's completely reasonable irrational prejudices"

Reply:
lol